user.js

```diff
@@ -32,6 +32,13 @@ user_pref("dom.webaudio.enabled",				false);
 // https://www.mozilla.org/en-US/firefox/geolocation/
 user_pref("geo.enabled",					false);
 
+// PREF: When geolocation is enabled, use Mozilla geolocation service instead of Google
+// https://bugzilla.mozilla.org/show_bug.cgi?id=689252
+user_pref("geo.wifi.uri", "https://location.services.mozilla.com/v1/geolocate?key=test");
```

Comment:
I had a look in FF55, same story. Yeah, I noticed the mozilla api key, it's used in

`browser.search.geoip.url`
https://location.services.mozilla.com/v1/country?key=%MOZILLA_API_KEY%